docs/setup/rke2-cluster.md

```diff
@@ -48,3 +48,47 @@ $ clusterctl describe cluster sample-cluster
     └─MachineDeployment/sample-cluster                    True                     22m
       └─Machine/sample-cluster-56df5b4499-zfljj           True                     23m
 ```
+
+## Deploying the Workload Cluster using SLEMicro
+
+1. Download the SLEMicro image from the Suse Customer Center and place it into the `{{ os_image_path }}` directory which is defined in the `./extra-vars.yml` file.
```

Comment:
We could link  https://www.suse.com/download/sle-micro/ here and mention its the 5.5 raw image that's required?

Reply:
Ok, makes sense I will change it to add the link and the required version.
Thanks Edu also for the naming issue ;)